src/Services/OData/ODataRequest.cs

```diff
@@ -16,53 +16,195 @@
+        /// </summary>
         public int Top { get; private set; }
+        /// <summary>
+        /// Gets the value of the "$skip" OData parameter if it exists, otherwise returns with 0.
+        /// </summary>
         public int Skip { get; private set; }
+        /// <summary>
+        /// Gets the value of the "$inlinecount" OData parameter if it exists, otherwise returns with "None".
+        /// </summary>
         public InlineCount InlineCount { get; private set; }
+        /// <summary>
+        /// Gets true if the last URI segment of the requested resource is "$count".
+        /// </summary>
         public bool CountOnly { get; private set; }
+        /// <summary>
+        /// Gets the value of the "$format" OData parameter if it exists, otherwise returns with null.
+        /// </summary>
         public string Format { get; internal set; }
+        /// <summary>
+        /// Gets the collection of <see cref="SortInfo"/> from the "$orderby" parameter of the webrequest.
+        /// </summary>
         public IEnumerable<SortInfo> Sort { get; internal set; }
+        /// <summary>
+        /// Gets the string list of the projection from the "$select" parameter of the webrequest.
+        /// </summary>
         public List<string> Select { get; private set; }
+        /// <summary>
+        /// Gets the string list of the expanded members from the "$expand" parameter of the webrequest.
+        /// </summary>
         public List<string> Expand { get; private set; }
+        /// <summary>
+        /// Gets the parsed <see cref="Expression"/> from the "$filter" parameter of the webrequest.
+        /// </summary>
         public Expression Filter { get; internal set; }
+        /// <summary>
+        /// Gets the value of the "enableautofilters" parameter of the webrequest if it exists, otherwise returs with FilterStatus.Default.
+        /// </summary>
         public FilterStatus AutofiltersEnabled { get; internal set; }
+        /// <summary>
+        /// Gets the value of the "enablelifespanfilter" parameter of the webrequest if it exists, otherwise returs with FilterStatus.Default.
+        /// </summary>
         public FilterStatus LifespanFilterEnabled { get; internal set; }
+        /// <summary>
+        /// Gets the value of the "queryexecutionmode" parameter of the webrequest if it exists, otherwise returs with FilterStatus.Default.
+        /// </summary>
         public QueryExecutionMode QueryExecutionMode { get; internal set; }
+        /// <summary>
+        /// Gets the value of the "metadata" parameter of the webrequest if it exists, otherwise returs with MetadataFormat.Full.
+        /// </summary>
         public MetadataFormat EntityMetadata { get; internal set; }
+        /// <summary>
+        /// Gets true if the "includebackurl" parameter of the webrequest is "true" if it exists, otherwise returs with false.

```

Comment:
Gets true if the "includebackurl" parameter of the webrequest exists and is "true", otherwise returns with false.

Reply:
ok